LandSandBoat/server
# [SQL][Items] Fix killer effects and charm effect

**_I affirm:_**

## What does this pull request do?

A few things.
1. Most of the BST armor only effects Killer Effects that the BST has natively, so it should not have undead, demon, dragon, etc. The ones BST does not have native have been removed.
2. I noticed while doing the above, many of the mods for charm+ were set as tame+ in the item_mods file. I went through every one with the tame+ modifier and cross checked them, correcting them.
3. I also noticed many of the BST gear that I was checking for one of the two above did not have item_pet_mods complete, even though most of them have pet mods. I added the missing pet mods.
4. Last, I fixed anything else about the armor I checked, and added any TODOs that was needed due to lack of mods (Reward recast decrease and augmentation).

## Steps to test these changes

As GM, add the item to your inventory through !additem ITEMID#
Equip the item and check the changes, making sure they match.
